src/benchmarks/micro/libraries/System.Numerics.Vectors/Perf_VectorOfT.cs

```diff
@@ -0,0 +1,183 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.Numerics.Tests
+{
+    public class Perf_VectorOfByte : Perf_VectorOf<byte> { }
+
+    public class Perf_VectorOfDouble : Perf_VectorOf<double> { }
+
+    public class Perf_VectorOfInt16 : Perf_VectorOf<short> { }
+
+    public class Perf_VectorOfInt32 : Perf_VectorOf<int> { }
+
+    public class Perf_VectorOfInt64 : Perf_VectorOf<long> { }
+
+    public class Perf_VectorOfSByte : Perf_VectorOf<sbyte> { }
+
+    public class Perf_VectorOfSingle : Perf_VectorOf<float> { }
+
+    public class Perf_VectorOfUInt16 : Perf_VectorOf<ushort> { }
+
+    public class Perf_VectorOfUInt32 : Perf_VectorOf<uint> { }
+
+    public class Perf_VectorOfUInt64 : Perf_VectorOf<ulong> { }
```

Comment:
Each benchmark takes something around 5 seconds to execute. The `Perf_VectorOf<T>` type defines 47 benchmarks so we need almost 4 minutes per one generic type. Here we have 10 generic types, which in total adds an extra 40 minutes. 

I know that it's important to benchmark more than one type, but perhaps we could limit it to two|three most important types?

Reply:
They can have vastly different perf characteristics and are independently handled in the runtime (sometimes even differing between signed/unsigned of the same size).

Is there a concept of inner/outer loop tests so we can test all on a less regular or explicit cadence?

Reply:
@tannergooding please excuse me for the delay. Can we agree on having just the 3 most common types?

Reply:
Is there a way I can exclude the other 7 types from running by default?

We really, especially as I start doing additional code improvements in this area, need the other benchmarks to be available for testing so improvements on this core type can be validated.

We use all of the primitive types, but particularly `byte`, `ushort`, and `int` to accelerate core workloads across the BCL, ASP.NET Core, and others: https://source.dot.net/#System.Private.CoreLib/Vector_1.cs,e1ed53562feae514,references

Reply:
@tannergooding Put the extra benchmarks in another type and don't put it in the Libraries category. Our runs only include Libraries and Runtime by default.